deployment/updates/scripts/add_permissions/2021_08_10_service_baseentry.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
rename file to deployment/updates/scripts/add_permissions/2021_08_10_baseentry_add_permission.php